content/20.treatments.md

```diff
@@ -349,24 +344,24 @@ Furthermore, high doses (1-3 g/day) significantly reduced the length of an ICU s
 Vitamin C also shortened the duration of mechanical ventilation by 18.2% in three trials in which patients required intervention for over 24 hours (95% CI 7.7% to 27%; p = 0.001)  [@doi:10.3390/nu11040708]. 
 Despite these findings, the CITRUS ALI study failed to show a benefit of a 96-hour infusion of vitamin C to treat ARDS, which is a severe complication of COVID-19 infection [@doi:10.1001/jama.2019.11825].
 
-###### Summary
+###### 4.3.b.iii.c Summary
 
 The evidence suggests that vitamin C supplementation can shorten the duration of a cold, reduce an individual's susceptibility to infections, and shorten a patient's stay in ICU when administered at high doses.
 While there is anecdotal evidence from China and the United States that vitamin C is currently being used for the treatment of COVID-19,  further controlled research is required.
 
-##### Vitamin D
+##### 4.3.b.iv Vitamin D
```

Comment:
Is there any way to generate these automatically (maybe tagging @agitter or @dhimmel here)? I think that maintaining these header identifiers by hand might turn out to be brutal. Having them done automatically can also help to show where a header level is not as intended.

Reply:
Pandoc's manual states:
![image](https://user-images.githubusercontent.com/3532898/79226821-9ca1e700-7e24-11ea-90c3-cf0b0a86dc6d.png)

I haven't tested this.  My reading is that automatic numbering is supported, in which case I definitely agree we should use it instead of manually updating these.  I'm not sure why only HTML is supported.  Hopefully because we're using athenapdf to convert the HTML manuscript to PDF, we would get the section numbers.

@rando2 if you'd like me to explore numbering, you can create a new issue and assign it to me to work on post-merge.